FraudProtection/src/main/java/com/paypal/android/fraudprotection/PayPalDataCollectorRequest.kt

```diff
@@ -0,0 +1,16 @@
+package com.paypal.android.fraudprotection
+
+/**
+ * Request object containing parameters to configure fraud protection data collection.
+ *
+ * @property [hasUserLocationConsent] true if the user has explicitly granted permission for a
+ * merchant app to collect device data to protect against fraud; false if explicit permission was
+ * not granted (default).
```

Comment:
```suggestion
 * @property [hasUserLocationConsent] informs the SDK if your application has obtained
 * consent from the user to collect location data in compliance with 
 * <a href="https://support.google.com/googleplay/android-developer/answer/10144311#personal-sensitive">Google Play Developer Program policies</a> 
 * This flag enables PayPal to collect necessary information required for Fraud Detection and Risk Management. 
```

To align with doc strings across SDKs